# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../xplanet/gcc44.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -Nur xplanet-1.2.1-orig/src/Satellite.cpp xplanet-1.2.1/src/Satellite.cpp
--- xplanet-1.2.1-orig/src/Satellite.cpp	2010-11-29 17:07:25.000054085 +0100
+++ xplanet-1.2.1/src/Satellite.cpp	2010-11-29 17:14:28.296055793 +0100
@@ -1,4 +1,5 @@
 #include <clocale>
+#include <cstdio>
 #include <cstring>
 #include <iostream>
 using namespace std;
diff -Nur xplanet-1.2.1-orig/src/Separation.cpp xplanet-1.2.1/src/Separation.cpp
--- xplanet-1.2.1-orig/src/Separation.cpp	2010-11-29 17:07:24.996053842 +0100
+++ xplanet-1.2.1/src/Separation.cpp	2010-11-29 17:14:40.492051707 +0100
@@ -1,4 +1,5 @@
 #include <cmath>
+#include <cstdio>
 #include <iostream>
 
 #include "Separation.h"
